test-cases/performance/perf-regression-latency-650gb-elasticity.yaml

```diff
@@ -34,6 +34,8 @@ use_hdrhistogram: true
 email_subject_postfix: 'elasticity test'
 nemesis_double_load_during_grow_shrink_duration: 30
 parallel_node_operations: true
+use_capacity_reservation: true
+instance_provision: 'on_demand'
```

Comment:
It won't help

The pipeline needs to be changed

Reply:
changed